src/reducers/auth.js

```diff
@@ -46,6 +46,10 @@ module.exports = (state, action) => {
 
         case StrangeAuth.types.LOGIN_FAIL:
 
+            if (payload.code === 'NO_TOKEN_ON_INIT') {
+                return state;
+            }
```

Comment:
Did this always error if the user started the app and attempted to login with no stored token? Thanks for this error code 👍